.github/workflows/feature_deploy.yml

```diff
@@ -0,0 +1,585 @@
+name: Feature environment deployment
+
+on:
+  push:
+    branches:
+      - feat/FN-72/deploy-app-code
+
+env:
+  environment: feature
+  credentials: ${{ secrets.AZURE_DIGITAL_DEV }}
+  from: latest
+  acr: ${{ secrets.ACR_REGISTRY_FEATURE }}
+  acr_username: ${{ secrets.ACR_USERNAME_FEATURE }}
+  acr_password: ${{ secrets.ACR_PASSWORD_FEATURE }}
+  # TODO: FN-72 NOTE: While we are using the dev GH environment, we can't use the RESOURCE_GROUP secret
+  # (It doesn't need to be a secret anyway.) So we define our resource group in an envioronment variable
+  # here and use that. The Azure region is the same so I've left that as using the secret.
+  resource_group: Digital-Feature
+
+jobs:
+  deployment-environment:
+    name: Environment setup
+    runs-on: [self-hosted, linux, deployment]
+    outputs:
+      environment: ${{ env.environment }}
+    steps:
+    - name: Initialise Environment
+      run: echo "Setting up ${{ env.environment }} environment"
+
+  #################### 1. PORTAL UI ####################
+  deploy-portal-ui:
+    needs: deployment-environment
+    name: Portal UI
+    runs-on: [self-hosted, linux, deployment]
+    # TODO: FN-72 NOTE: we don't currently have a `feature` GitHub environment yet so use the `dev` one while we're not doing much.
+    # Restore the parameterised version throughout!
+    environment: dev
+    # environment:
+    #   name: ${{ needs.deployment-environment.outputs.environment }}
+    env:
+      repository: portal-ui
+      
+    steps:
+    - name: 1. Latest commit hash
+      uses: actions/checkout@v3
+
+    - name: 2. Build and push
+      uses: docker/build-push-action@v1
+      with:
+        registry: ${{ env.acr }}
+        username: ${{ env.acr_username }}
+        password: ${{ env.acr_password }}
+        repository: ${{ env.repository }}
+        path: portal
+        tags: latest
+        build_args: GITHUB_SHA=${{ github.sha }}
+        add_git_labels: true
+
+    - name: 3. ACR login
+      uses: azure/docker-login@v1
+      with:
+        login-server: ${{ env.acr }}
+        username: ${{ env.acr_username }}
+        password: ${{ env.acr_password }}
+
+    - name: 4. Tag images
+      run: |
+        from=${{ env.from }}
+        to=${{ env.environment }}
+        docker pull ${{ env.acr }}/${{ env.repository }}:$from
+        docker tag ${{ env.acr }}/${{ env.repository }}:$from ${{ env.acr }}/${{ env.repository }}:$to
+        docker push ${{ env.acr }}/${{ env.repository }}:$to
+
+    - name: 5. Azure login
+      uses: azure/login@v1
+      with:
+        creds: ${{ env.credentials }}
+
+    - name: 6. Slot
```

Comment:
Note that it failed when I just used the same approach as the other deploy scripts - setting the default value for the resource group. When deploying multiple services at the same time, it seemed to not preserve the default resource group. Odd. Anyway, resolvable by putting it in the commands.

I suspect they have issues too?